python/doc/examples/meta_modeling/chaos_beam_sensitivity_degree.ipynb

```diff
@@ -0,0 +1,568 @@
+    "    enumfunc = multivariateBasis.getEnumerateFunction()\n",
+    "    P = enumfunc.getStrataCumulatedCardinal(totalDegree)\n",
+    "    # Get number of coefficients in the selection\n",
+    "    indices = chaosResult.getIndices()\n",
+    "    nbcoeffs = indices.getSize()\n",
+    "    # Compute rate\n",
+    "    sparsityRate = 1.0 - nbcoeffs / P\n",
+    "    return sparsityRate"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following functions compute and plots the Q2 predictivity coefficients within the validation plot."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def computeQ2Chaos(chaosResult, inputTest, outputTest):\n",
+    "    \"\"\"Compute the Q2 of a chaos.\"\"\"\n",
+    "    metamodel = chaosResult.getMetaModel()\n",
+    "    val = ot.MetaModelValidation(inputTest, outputTest, metamodel)\n",
+    "    Q2 = val.computePredictivityFactor()\n",
+    "    Q2 = max(Q2, 0.0)  # We are not lucky every day.\n",
+    "    return Q2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def printChaosStats(multivariateBasis, chaosResult, inputTest, outputTest, totalDegree):\n",
+    "    \"\"\"Print statistics of a chaos.\"\"\"\n",
+    "    sparsityRate = computeSparsityRate(multivariateBasis, totalDegree, chaosResult)\n",
+    "    Q2 = computeQ2Chaos(chaosResult, inputTest, outputTest)\n",
+    "    metamodel = chaosResult.getMetaModel()\n",
+    "    val = ot.MetaModelValidation(inputTest, outputTest, metamodel)\n",
+    "    graph = val.drawValidation()\n",
+    "    legend1 = \"D=%d, Q2=%.2f%%\" % (totalDegree, 100 * Q2)\n",
+    "    graph.setLegends([\"\", legend1])\n",
+    "    graph.setLegendPosition(\"topleft\")\n",
+    "    print(\n",
+    "        \"Degree=%d, Q2=%.2f%%, Sparsity=%.2f%%\"\n",
+    "        % (totalDegree, 100 * Q2, 100 * sparsityRate)\n",
+    "    )\n",
+    "    return graph"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "multivariateBasis = ot.OrthogonalProductPolynomialFactory(\n",
+    "    [dist_E, dist_F, dist_L, dist_I]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "N = 20  # size of the train design"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "n_valid = 1000  # size of the test design"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The seed is selected to get *interesting* results. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "magicSeed = 43  # 127 is funny too\n",
```

Comment:
"funny" in what way? Is the user supposed to execute the notebook with 127 to see what changes?